rewrite-gradle/src/main/java/org/openrewrite/gradle/GradleExecutionContextView.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.gradle;
+
+import org.openrewrite.DelegatingExecutionContext;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.ipc.http.HttpUrlConnectionSender;
+
+public class GradleExecutionContextView extends DelegatingExecutionContext {
```

Comment:
```suggestion
public class HttpSenderExecutionContextView extends DelegatingExecutionContext {
```

This view isn't really Gradle-specific. All Recipes which need to issue HTTP requests, including Maven recipes, will be updated to get their `HttpSender` from execution context

Reply:
Renamed and moved this into rewrite-core.